packages/venia-ui/lib/components/CartPage/GiftCards/giftCard.js

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+
+import { useGiftCard } from '@magento/peregrine/lib/talons/CartPage/GiftCards/useGiftCard';
+
+import { mergeClasses } from '../../../classify';
+import defaultClasses from './giftCard.css';
+
+const GiftCard = props => {
+    const {
+        // appliedBalance,
+        code,
+        // currentBalance,
+        // expirationDate,
```

Comment:
I can remove these here and from the query, they are available but aren't currently used.